lib/services/devices/deviceService.js

```diff
@@ -260,11 +260,21 @@ function mergeDeviceWithConfiguration(fields, defaults, deviceData, configuratio
 
```

Comment:
CHANGE_NEXT_RELEASE entry (mentioning #569) missing

Reply:
Fixed in 63fe182ea98e6280914a7c6a4366550762f8a636